admin/local/cli/qserv

```diff
@@ -37,15 +37,15 @@ from lsst.qserv.admin.qservCli.qserv_log import log_level_from_args
 from lsst.qserv.admin.qservCli.subproc import QservSubprocessError
 
 if __name__ == "__main__":
-    success, levelMsg = log_level_from_args(sys.argv)
+    success, level_msg = log_level_from_args(sys.argv)
```

Comment:
Just for my own education. Did you migrate this name to stick to the "all-underscores" naming convention, or `ruff` did it?

Reply:
I probably let ruff make that change.  I used it's auto-editors as much as possible, since they are generally very principled about what they will auto-edit, and what the will leave to you as "possibly dangerous."